cmd/minishift/cmd/completion.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
2017? And 'Red Hat\'? Or you want to keep the original since you copied from Minikube?